packages/eas-cli/src/commandUtils/context/contextUtils/getProjectIdAsync.ts

```diff
@@ -127,29 +128,37 @@ export async function validateOrSetProjectIdAsync({
       );
     }
 
-    const actorUsername = getActorUsername(actor);
-    if (!exp.owner && appForProjectId.ownerAccount.name !== actorUsername) {
-      if (actorUsername) {
-        throw new Error(
-          `Project config: Owner of project identified by "extra.eas.projectId" (${
-            appForProjectId.ownerAccount.name
-          }) does not match the logged in user (${actorUsername}) and the "owner" field is not specified. To ensure all libraries work correctly, "owner": "${
-            appForProjectId.ownerAccount.name
-          }" should be added to the project config, which can be done automatically by re-running "eas init". ${learnMore(
-            'https://expo.fyi/eas-project-id'
-          )}`
-        );
-      } else {
-        // robot caller
-        throw new Error(
-          `Project config: Owner of project identified by "extra.eas.projectId" (${
-            appForProjectId.ownerAccount.name
-          }) must be specified in "owner" field when using a robot access token. To ensure all libraries work correctly, "owner": "${
-            appForProjectId.ownerAccount.name
-          }" should be added to the project config, which can be done automatically by re-running "eas init". ${learnMore(
-            'https://expo.fyi/eas-project-id'
-          )}`
-        );
+    const sdkVersion = exp.sdkVersion;
+    if (
+      sdkVersion &&
+      (semver.satisfies(sdkVersion, '>= 53.0.0') || sdkVersion.includes('canary'))
+    ) {
+      // SDK 53 and above no longer require owner field in app config
+    } else {
```

Comment:
Suggestion: structure the code so that old SDK versions are handled more like an edge case than a default, such as when `sdkVersion` is not defined:
```suggestion
    const sdkVersion = exp.sdkVersion;
    if (sdkVersion && semver.satisfies(sdkVersion, '< 53.0.0')) {
```
53.0.0-canary will not enforce the owner field, but 52.0.0-canary will.